client/gutenberg/extensions/videopress/editor.js

```diff
@@ -43,8 +48,41 @@ const addVideoPressSupport = ( settings, name ) => {
 				default: 'metadata',
 			},
 		},
+
+		transforms: {
+			...settings.transforms,
+			from: [
+				{
+					type: 'files',
+					isMatch: files => every( files, file => file.type.indexOf( 'video/' ) === 0 ),
+					// We define a higher priority (lower number) than the default of 10. This ensures that this
+					// transformation prevails over the core video block default transformations.
+					priority: 9,
```

Comment:
Thanks for the comment here! It's saves us from having to search for documentation on magic numbers.